.travis.yml

```diff
@@ -2,6 +2,44 @@ language: java
 
 sudo: true
 
+addons:
+  apt:
+    packages:
+      - texlive-fonts-recommended
+      - texlive-latex-recommended
+      - otf-freefont
+      - gsfonts
+      - gsfonts-other
+      - gsfonts-x11
+      - ttf-mscorefonts-installer
+      - t1-xfree86-nonfree
+      - ttf-alee
+      - ttf-ancient-fonts
+      - ttf-arabeyes
+      - ttf-arphic-bkai00mp
+      - ttf-arphic-bsmi00lp
+      - ttf-arphic-gbsn00lp
+      - ttf-arphic-gkai00mp
+      - ttf-atarismall
+      - ttf-bpg-georgian-fonts
+      - ttf-dustin
+      - ttf-f500
+      - ttf-sil-gentium
+      - ttf-georgewilliams
+      - ttf-isabella
+      - ttf-larabie-deco
+      - ttf-larabie-straight
+      - ttf-larabie-uncommon
+      - ttf-sjfonts
+      - ttf-staypuft
+      - ttf-summersby
+      - fonts-ubuntu-title
+      - ttf-xfree86-nonfree
+      - xfonts-intl-european
+      - xfonts-jmk
+      - xfonts-terminus
```

Comment:
On integration tests, there is some differences between travis and local execution. I think some fonts are missing on travis trusty instance. So I try to install almost all fonts to see if it changes something.
I will fix package list to fit trusty apt repo.